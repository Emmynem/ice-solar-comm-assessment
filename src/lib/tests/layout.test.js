import React from "react";
import renderer from "react-test-renderer";
import Layout from "../layout";

describe("Layout", () => {
    it("renders properly", () => {
        const tree = renderer
            .create(<Layout />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});