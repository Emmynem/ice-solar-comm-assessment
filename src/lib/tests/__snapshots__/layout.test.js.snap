// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders properly 1`] = `
<div
  className="container mt-5"
>
  <div>
    <h1>
      Login
    </h1>
  </div>
  <div
    className="card mt-5 mb-5"
  >
    <div
      className="card-header"
    >
      Fake Store
    </div>
    <div
      className="card-body"
    >
      <form>
        <div
          className="form-group"
        >
          <div
            className="row"
          >
            <div
              className="col-12 col-lg-6"
            >
              <label
                htmlFor="username"
              >
                Username:
              </label>
              <input
                className="form-control mt-2"
                id="username_login"
                name="username"
                onChange={[Function]}
                placeholder="Username"
                type="text"
                value=""
              />
              <p
                className="text-danger"
                placeholder="err_name"
              />
            </div>
            <div
              className="col-12 col-lg-6"
            >
              <label
                htmlFor="password"
              >
                Password:
              </label>
              <input
                className="form-control mt-2"
                id="password_login"
                name="password"
                onChange={[Function]}
                placeholder="Enter password"
                type="password"
                value=""
              />
              <p
                className="text-danger"
                placeholder="err_word"
              />
            </div>
          </div>
          <div
            className="clearfix"
          >
            <button
              className="btn btn-success mt-3"
              onClick={[Function]}
              type="submit"
            >
              Login
            </button>
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
